model/qti/Service.php

```diff
@@ -237,4 +237,37 @@ public function hasItemModel(core_kernel_classes_Resource $item, $models)
     {
         return taoItems_models_classes_ItemsService::singleton()->hasItemModel($item, $models);
     }
+
+    /**
+     * Delete the contents of the item, but not the resource representing it.
+     *
+     * @param core_kernel_classes_Resource $item
+     * @return bool
+     */
+    public function deleteContentByRdfItem(core_kernel_classes_Resource $item)
+    {
+        return taoItems_models_classes_ItemsService::singleton()->deleteItemContent($item);
+    }
+
+    public function backupContentByRdfItem(core_kernel_classes_Resource $item)
```

Comment:
A backup should keep the current directory, isn't it? Here you are removing the original dir because it will not exist after this operation.